doc-surrealdb_versioned_docs/version-latest/surrealql/statements/define/access/jwt.mdx

```diff
@@ -236,3 +238,42 @@ The database token payload should at least include the following claims when use
   "db": "app_vitalsense"
 }
 ```
+
+## With `AUTHENTICATE` clause 
+
+The `AUTHENTICATE` clause allows you to define a custom expression that will be executed when the token is verified. This expression will be executed in the context of the token, allowing you to perform additional checks on the token claims before the token is accepted. If the expression returns `false`, the token will be rejected.
+
+#### Example: Using an HMAC algorithm with an `AUTHENTICATE` clause.
```

Comment:
As mentioned via DM, I would replace this example by [the one in the tests](https://github.com/surrealdb/surrealdb/pull/4385/files#diff-8ac7635faed5d17317e3baf51f02e1fa8ce69d80cf45aee5dcfa381f62ce1640R2255-R2261).

Reply:
The reason is that expiration cannot be checked in the `AUTHENTICATE` clause as SurrealDB already rejects expired tokens before.